src/ensembl/src/content/home/Home.scss

```diff
@@ -1,44 +1,223 @@
 @import 'src/styles/common';
 
 .home {
-  padding-bottom: 80px;
+  display: grid;
+  grid-template-rows: minmax(550px, auto) 85px;
+}
 
-  h2 {
-    color: $grey;
-    font-size: 14px;
-    font-weight: $normal;
-  }
+.container {
+  display: grid;
+  grid-template-rows: auto 1fr;
+  background-color: $blue;
+  background-image: linear-gradient($black, $blue);
+  padding-top: 40px; 
+}
+
+.middleHomeWrapper {
   position: relative;
 }
 
-.search {
-  background: $light-grey;
-  box-shadow: 0 3px 5px $medium-light-grey;
-  min-height: 235px;
-  padding: 15px 60px;
+.middleHome {    
+  display: grid;
+  grid-template-rows: 38px 110px 100px; 
+  justify-content: center;
+  height: 248px;
+
+  .logotypeContainer {
```

Comment:
Why does this class exist? You can remove it entirely and add `justify-self: center` to the `.logotype`

Reply:
I tried this before but it didn't work. Maybe because its an svg and we cant center the svg itself but if you center the parent content then it works.

Reply:
sorry ignore above comment, turn out i need to remove the parent completely and it works with justify-self.